packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -412,18 +423,26 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   }, [uploads]);
 
   const onSendMessageHandler = useCallback(
-    async function (content: string, /* @conditional-compile-remove(attachment-upload) */ options?: MessageOptions) {
+    async function (
+      content: string,
+      /* @conditional-compile-remove(attachment-upload) */ /* @conditional-compile-remove(rich-text-editor-composite-support) */ options?: MessageOptions
+    ) {
       /* @conditional-compile-remove(attachment-upload) */
       const attachments = options?.attachments ?? [];
       /* @conditional-compile-remove(attachment-upload) */
       handleUploadAction({ type: AttachmentUploadActionType.Clear });
       /* @conditional-compile-remove(attachment-upload) */
       await adapter.sendMessage(content, {
-        attachments: attachments
+        attachments: attachments,
+        /* @conditional-compile-remove(rich-text-editor-composite-support) */
+        type: options?.type
```

Comment:
Just to confirm, have we checked that his work correctly in stable (that adapter func is called with empty object as a second parameter)?

Reply:
Confirmed no issues